lib/workers/repository/extract/index.ts

```diff
@@ -47,6 +47,17 @@ export async function extractAllDependencies(
   const extractResults = await Promise.all(
     extractList.map(async (managerConfig) => {
       const packageFiles = await getManagerPackageFiles(managerConfig);
+      if (packageFiles) {
+        for (const p of packageFiles) {
```

Comment:
```suggestion
        for (const p of packageFiles ?? []) {
```
can we move the condition to the loop, so we've less nesting?

can you do same for deps array? but I'm pretty sure deps is never null or undefined. 🤔

Reply:
I was thinking the same thing✌
There was a use-case for the if condition for deps, will link it here soon when I recall it.